### ISSUE TITLE
A11y_VSCode_ Integrated Terminal_Usability: Unecessary decorative content announced for screen reader user.

### ISSUE THREAD

**Issue by RohitJadhav-TCS** [microsoft/vscode#76867]
**Environment Details:**
VSCode Version : 1.36.0 (user setup)
OS Version : Win10

**Additional Details:**
Usability
ScreenReader Versions : NVDA2019.1.1, JAWS 2019

**Repro Steps:**
1.Open Visual studio code editor.
2.Using "Ctrl+`" key open the Integrated Terminals.
3.Navigate to terminal  action toolbar.
4.Navigate to combo box and expand it using alt+down arrow key.
5.Navigate to separator.

**Actual:**
When screen reader users navigate to the combo box and separator, it is announcing it as part of list.
Unnecessary decorative content is announced for screen reader user.

This issue has been observed in "Win 10 + JAWS" and "Win10 + NVDA".

**Expected:**
Screen reader focus should not go on separator and it should not announce anything for separator. 

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
Screen reader user will get confused when it announce separator as part of list.

**Attachment for Reference:**
[VSCode_ IntegratedTerminal_Usability Unecessary decorative content announced for screen reader user.pptx](https://github.com/microsoft/vscode/files/3368006/VSCode_.IntegratedTerminal_Usability.Unecessary.decorative.content.announced.for.screen.reader.user.pptx)

**Comment by Tyriar** [comment]
Please don't create duplicate issues for multiple OSs

/duplicate https://github.com/microsoft/vscode/issues/76878
